package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "scripts": {
     "test": "jest",
-    "start": "NODE_ENV=development nodemon app.js",
+    "start": "NODE_ENV=development node app.js",
```

Comment:
any reason why you changed this from `nodemon` to `node`?

Reply:
No reason, I thought `node` was being used in the `package.json` on github. I should've checked before pushing this one 👎